### PR TITLE
 OhemLoss: bug fix

### PR DESCRIPTION
**Pull request by danielafrimi**
OhemLoss forward, if all pixels are ignore labels, return empty loss tensor and pass it to logits device

### CODE REVIEW

**Review by BloodAxe — APPROVED** [review]
LGTM
